### PR TITLE
Create internship-expense-policy.md

### PR DESCRIPTION
**Pull request by TQuock**
adding an internship expense policy section to the handbook

### CODE REVIEW

File: operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Internship Expense Policy
+
+At Mattermost, we want our interns to be successful from Day 1 and you'll be provisioned an Expensify account. As a general rule, our [expense policy](https://handbook.mattermost.com/operations/finance/spending-company-money/expense-policy) for our full-time and part-time staff members is the same for our interns with some modifications. As an intern, you can refer to our How to spend company money policy for guidance for most expense-related questions.
```
Comment (justinegeffen) [inline operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md:3]: The "expense policy" URL isn't working. Is it still to be added? Or will it be this one: https://github.com/mattermost/mattermost-handbook/pull/501/?

The "How to spend company money" doesn't have a link. Will a link to the new page be added or should we reference the current one?

File: operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The purpose of this section is to identify the variance from our full time staff member spending policy.
+
+## Notable spending differences for interns
+
+In general, we are a culture where you don’t have to ask permission before making purchases on behalf of the company unless there are guidelines specific on what and how much can be purchased. If in doubt, ask your manager before you make any purchases to confirm you are following the purchasing guidelines. A good reference point for spending is the [expense policy](https://handbook.mattermost.com/operations/finance/spending-company-money/expense-policy) section if you are uncertain.
```
Comment (justinegeffen) [inline operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md:9]: As per above: The "expense policy" URL isn't working. Is it still to be added? Or will it be this one: #501?

**Review by justinegeffen — APPROVED** [review]

File: operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md (hunk 1 above)
Comment (TQuock) [reply operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md:3]: @justinegeffen that is correct that #501 is the PR for the expense policy page. The How to Spend Company Money will be migrated to the Expense Policy Page.

File: operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md (hunk 2 above)
Comment (TQuock) [reply operations/finance/spending-company-money/staff-member-expenses/internship-expense-policy.md:9]: Please see above comment

**Review by ialorro — APPROVED** [review]
looks good!

**Comment by justinegeffen** [comment]
@TQuock, I'd like to propose holding off on merging these for now. I'm concerned that these pages will link to pages that haven't yet been merged which may cause a frustrating user experience. There's also a possibility that existing pages may become unavailable which we'd like to avoid.

One option is to put together a very brief outline of the pages that are being deprecated with their URL, their replacements with their URL, and the overall structure that you'd like so that we can make sure that all the deprecated pages have redirects so that in the event that the ``gitbook.yaml`` idea takes longer than anticipated, we have a fallback. 

I know the current structure isn't ideal hence the work you've put in on all of this. But I'd like to make sure it's easier going forward. :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**